src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,38 @@
+import { Status } from '../../types/Status';
+import { Todo } from '../../types/Todo';
+import { TodoFilter } from '../TodoFilter/TodoFilter';
+
+type Props = {
+  completedTodos: Todo[],
+  notCompletedTodos: Todo[],
```

Comment:
footer does not need to have the arrays of todos, let's pass only the length here
```suggestion
  nrOfCompletedTodos:  number,
  nrOfNotCompletedTodos: number,
```